File: eCinema/eCinema.Model/Korisnik.cs
namespace eCinema.Model
{
    public class Korisnik
    {
        public int KorisnikId { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string KorisnickoIme { get; set; }
        public string Email { get; set; }
        public string BrTelefona { get; set; }
        public bool Active { get; set; }
        public virtual ICollection<KorisnikUloge> KorisnikUloges { get; set; }

        public string imePrezime => $"{Ime} {Prezime}";
        public override string ToString()
        {
            return KorisnickoIme;
        }
    }
}

File: ecinema_mobile/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? ime;
  String? prezime;
  String? korisnickoIme;
  String? email;
  String? brTelefona;
  String? lozinka;

  Korisnik(
      {this.korisnikId,
      this.ime,
      this.prezime,
      this.korisnickoIme,
      this.email,
      this.brTelefona,
      this.lozinka});

  factory Korisnik.fromJson(Map<String, dynamic> json) =>
      _$KorisnikFromJson(json);
  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}
